webapp/first_snap/views.py

```diff
@@ -2,13 +2,12 @@
 import flask
 import re
 from io import StringIO
-from ruamel.yaml import YAML
 
+from webapp import helpers
 from webapp.first_snap import logic
 
 YAML_KEY_REGEXP = re.compile(r"([^\s:]*)(:.*)")
 
-yaml = YAML()
```

Comment:
@jpmartinspt In FSF we use `YAML()` instead of `YAML(typ="safe")`. The default `YAML()` config allows to preserve original yaml file formatting that is used to render back snapcraft.yaml with correct formatting.

I guess we should use `YAML()` for all our yaml reading or allow passing a `typ` argument to `read_yaml` to override it. As I understand using `YAML(typ='rt')` is same as `YAML()`: https://yaml.readthedocs.io/en/latest/basicuse.html